perceval/utils/statevector.py

```diff
@@ -45,6 +45,7 @@
 import sympy as sp
 
 from exqalibur import FockState, FSArray
+from perceval.utils.qmath import exponentiation_by_squaring
```

Comment:
When you import from the same package, it's better to use relative imports in order to avoid potential circular dependencies:
`from .qmath import exponentiation_by_squaring`